api/SVGAltGlyphElement.json

```diff
@@ -6,35 +6,31 @@
         "spec_url": "https://www.w3.org/TR/SVG11/text.html#InterfaceSVGAltGlyphElement",
         "support": {
           "chrome": {
-            "version_added": "1"
+            "version_added": "1",
+            "version_removed": "40"
           },
           "chrome_android": "mirror",
           "edge": "mirror",
           "firefox": {
             "version_added": "4",
+            "version_removed": "49",
             "notes": "partial support, see <a href='https://bugzil.la/456286'>bug 456286</a> and <a href='https://bugzil.la/571808'>bug 571808</a>."
           },
           "firefox_android": "mirror",
           "ie": {
             "version_added": false
           },
           "oculus": "mirror",
-          "opera": {
-            "version_added": "10.6"
-          },
-          "opera_android": {
-            "version_added": "11"
-          },
+          "opera": "mirror",
```

Comment:
This will change `version_added` to "15". Wasn't this in 10.6 as the previous data claims?

Reply:
It was not supported before Opera 15, no; running the test in the collector on Opera 12.16 with https://github.com/foolip/mdn-bcd-collector/pull/2301 shows that the object is an `SVGElement` object, AKA a generic object.

Reply:
That checks out. I also confirmed that `<altGlyph>` parses to an `SVGElement`.

Can you update `svg/elements/altGlyph.json` as well?